hw/ip/aes/dv/aes_sim_cfg.hjson

```diff
@@ -61,6 +61,11 @@
       name: aes_sanity
       uvm_test: aes_sanity_test
       uvm_test_seq: aes_stress_vseq
+    }
+     {
```

Comment:
Tiny nit: Spacing (line 65 has one too many spaces?).

Also, if you're touching this commit, there's a typo in the title (should be "key_length", I think)

Reply:
fixed